vscode-lean4/src/leanclient.ts

```diff
@@ -1,36 +1,20 @@
 import { TextDocument, Position, TextEditor, EventEmitter, Uri, Diagnostic,
     workspace, languages, DocumentHighlight, Range, DocumentHighlightKind, window, Disposable } from 'vscode'
 import {
+    DidChangeTextDocumentParams,
+    DidCloseTextDocumentParams,
     DidOpenTextDocumentNotification,
     DocumentFilter,
     LanguageClient,
     LanguageClientOptions,
     Protocol2CodeConverter,
+    PublishDiagnosticsParams,
     ServerOptions,
-    VersionedTextDocumentIdentifier
 } from 'vscode-languageclient/node'
 import * as ls from 'vscode-languageserver-protocol'
 import { executablePath, addServerEnvPaths, serverLoggingEnabled, serverLoggingPath, getElaborationDelay } from './config'
-import { PlainGoal, PlainTermGoal, ServerProgress } from './leanclientTypes'
 import { assert } from './utils/assert'
-
-export interface LeanFileProgressProcessingInfo {
-    /** Range which is still being processed */
-    range: ls.Range;
-}
-
-export interface LeanFileProgressParams {
-    /** The text document to which this progress notification applies. */
-    textDocument: VersionedTextDocumentIdentifier;
-
-    /**
-     * Array containing the parts of the file which are still being processed.
-     * The array should be empty if and only if the server is finished processing.
-     */
-    processing: LeanFileProgressProcessingInfo[];
-}
-
-const processingMessage = 'processing...'
+import { PlainGoal, PlainTermGoal, LeanFileProgressParams } from '@lean4/infoview';
```

Comment:
This import doesn't work for me (red squiggles from tsserver).  I've run `npx lerna bootstrap`.

Reply:
Hmm, I'm not quite sure. Which version of NPM are you using? I'm on 7.19.0. I can check with another one. There was a separate ESLint directory issue but TS proper should work.

Reply:
I'm using 7.18 here.

Reply:
That should definitely work. Do you have `node_modules/@lean4/infoview/` in `vscode-lean4/`? It should contain a symlink to the infoview directory.

Reply:
Yes, I have that directory.

eslint is complaining about a missing tsconfig though, could this be a problem for typescript as well?
```
Parsing error: Cannot read file '/home/gebner/vscode-lean4/tsconfig.json'.
```

Reply:
Hah! It works once you run `npm run build`.

Reply:
Oh yeah, the infoview `dist/` has to be available. I'll add that to the README.